qa/L0_client_cancellation/client_cancellation_test.py

```diff
@@ -54,6 +54,7 @@ def callback(user_data, result, error):
     else:
         user_data._completed_requests.put(result)
 
+
```

Comment:
nit: consistent spacing between functions? Or is this something the autoformatter did